subsys/bluetooth/controller/ll_sw/ull_llcp_chmu.c

```diff
@@ -111,6 +106,8 @@ static void lp_chmu_ntf(struct ll_conn *conn, struct proc_ctx *ctx)
 
 static void lp_chmu_complete(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t evt, void *param)
 {
+	ull_conn_chan_map_set(conn, ctx->data.chmu.chm);
```

Comment:
I was thinking on removing that state at all from this procedure,   initially I though it will be used for updating connection object but it is not (and that procedure is not generating any HCI events)

Reply:
You're correct - didn't realize that.